runtime-sdk/modules/evm/src/backend.rs

```diff
@@ -0,0 +1,203 @@
+//! EVM backend.
```

Comment:
Could move the file rename in a separate commit, so that the actual changes can be reviewed more easily